modules/ROOT/pages/rtm-agent-proxy-config.adoc

```diff
@@ -0,0 +1,117 @@
+
+If your Mule runtime engine runs inside a firewall that restricts external communication through a proxy server, you can configure Runtime Manager agent to route traffic through the proxy server to Runtime Manager, and optionally, encrypt the authentication password.
+
+Runtime Manager agent supports only the basic access authentication method.
+
+
+== Configure the Agent to Connect Through a Proxy Server 
+
+Running the `amc_setup` command creates the `$MULE_HOME/conf/mule-agent.yml` file. 
+
+If you have previously installed Runtime Manager agent and want to change the configuration to use a proxy, you can add the proxy configuration to the `mule-agent.yml` file.
+
+To configure the proxy server connection, run this command:
+
+[source,console,linenums]
+----
+$MULE_HOME/bin/amc_setup -H token server-name -P proxy-host proxy-port proxy-user proxy-password
+----
+
+If the proxy server doesn't require authentication, omit _proxy-user_ and _proxy-password_.
+
+[%header,cols="20a,80a"]
+|===
+|Parameter|Description
+|_proxy-host_ a|Specifies the hostname of the desired proxy server, for example, `proxy.acme.com`.
+Do not include `http://` or `https://` in the hostname.
+|_proxy-port_|Specifies the port of the desired proxy server 
+|_proxy-user_ |(Optional) Specifies the user with which to authenticate against the proxy, if required
+|_proxy-password_ | (Optional) Specifies the password for the authentication _proxy-user_, if required
+
+|===
+
+=== Example amc_setup command lines
+
+This example configures the agent to work with a proxy server (`acme.proxy.com`) and specifies a Runtime Manager token:
+
+[source, console,linenums]
+----
+amc_setup -H myToken myMuleServer -P acme.proxy.com 443
+----
+
+This example configures the agent to work with a proxy server that requires authentication:
+
+[source,console,linenums]
+----
+amc_setup -H myToken myMuleServer -P acme.proxy.com 443 internalAdmin Ins1d3V0icePassword
+----
+
+
+[[encrypt_password]]
+== Include an Encrypted Password in the mule-agent.yml File
+
+You can encrypt the proxy server authentication password to use in the `mule-agent.yml` file.
+
+To include an encrypted password in your `mule-agent.yml` file:
+
+. Generate the encrypted password:
++
+`$MULE_HOME/bin/amc_setup --encrypt`
++
+The encryption utility prompts you for the master password and phrase. The master password is used to decrypt the phrase.
+. Enter the master password and phrase.
++
+The encryption utility outputs the encrypted phrase, for example:
++
+`"!PBEWITHSHA1ANDDESEDE,CmDci1pRMI+1gH89akTNFb/EjZ26pfIX"`
+. Edit the `$MULE_HOME/conf/mule-agent.yml` file.
++
+Replace the value of `password` in the `proxyConfiguration` element with the encrypted phrase from step 2. For example:
++
+[source,yaml,linenums]
+----
+globalConfiguration:
+  ...
+  authenticationProxy:
+    endpoint: https://arm-auth-proxy.prod.cloudhub.io
+  proxyConfiguration:
+    *password: "![PBEWITHSHA1ANDDESEDE,CmDci1pRMI+1gH89akTNFb/EjZ26pfIX]"*
```

Comment:
Looking at the editor, since this `password: "![PBEWITHSHA1ANDDESEDE,CmDci1pRMI+1gH89akTNFb/EjZ26pfIX]"` is inside a yaml, it's not being rendered as bold. I would remove the `*` since they are confusing
<img width="1020" alt="Screen Shot 2019-10-07 at 11 54 58" src="https://user-images.githubusercontent.com/2117502/66322933-54e05700-e8f9-11e9-9189-c26056177946.png">